### PR TITLE
Generate parameters.yml which is supported by new Install script 

### PR DESCRIPTION
**Pull request by vidarl**
This PR adds the possibility to generate a parameters.yml which can be used to setup ezpublish using the new install script ( php ezpublish/console ezplatform:install )

FYI : ezplatform:install is still WIP, and yet not merged to master

### CODE REVIEW

**Comment by vidarl** [comment]
@andrerom : Have a look. But I suggest we don't merge this yet, as "ezplatform:install" is WIP

File: README.md [hunk 1]
```diff
@@ -189,6 +189,17 @@ fig.sh is a wrapper for fig which also do some internal provisioning. Any comman
 
 When the containers are created, you should be able to browse to eZ Publish setup wizard by going to http://[VM_IP_ADDR]:8080/ if using vagrant and http://localhost:8080/ if running the containers on localhost 
 
+### Setting up eZ Publish using install script
+
+It is possible to setup a fresh installation using the install script instead of using the setup wizard.
+ - Set ```EZ_GENERATE_PARAMETERS=true``` in fig.config.
```
Comment (andrerom) [inline README.md]: Do we need a parameter for this? does it hurt to generate this on eZ Publish install?

Comment (vidarl) [inline README.md]: I guess it do not hurt. 
But I believe that when a parameter.yml is created by composer. So the container will overwrite that one.... 
But AFAIK, the user may generate that one again from parameter.yml-dist using "composer run-script post-update-cmd"

Comment (andrerom) [inline README.md]: composer (or rather `Incenteev\\ParameterHandler\\ScriptHandler::buildParameters`) only asks-about/generates parameters that are missing, as in parameters in `parameter.yml.dist` but not in `parameter.yml`

**Comment by andrerom** [comment]
installer is merged on friday, which is why devdemo is down.
checked PR against what was merged, and ready for review?

File: dockerfiles/ezpublish/install/run.sh [hunk 2]
```diff
@@ -189,5 +190,5 @@ if [ aa$EZ_ENVIRONMENT != "prod" ]; then
 fi
 
 php ezpublish/console assets:install --symlink --relative --env $EZ_ENVIRONMENT
-php ezpublish/console ezpublish:legacy:assets_install --symlink --relative --env $EZ_ENVIRONMENT
+#php ezpublish/console ezpublish:legacy:assets_install --symlink --relative --env $EZ_ENVIRONMENT
```
Comment (andrerom) [inline dockerfiles/ezpublish/install/run.sh]: Maybe something like a `if [ -d ezpublish_legacy ]` around it?

File: dockerfiles/ezpublish/install/run.sh [hunk 3]
```diff
@@ -175,12 +175,13 @@ if [ "$EZ_PACKAGEURL" != "" ]; then
   /install_packages.sh
 fi
 
+/generate_parameters_file.sh
 
 echo "Setting permissions on eZ Publish folder as they might be broken if rsync is used"
 sudo setfacl -R -m u:$APACHE_RUN_USER:rwx -m u:`whoami`:rwx \
-     ezpublish/{cache,logs,config,sessions} ezpublish_legacy/{design,extension,settings,var} web
```
Comment (andrerom) [inline dockerfiles/ezpublish/install/run.sh]: same here maybe? (so well as around the kickstart ini code which seems to be untouched in this PR)

**Comment by vidarl** [comment]
@andrerom : ready for review now

File: dockerfiles/ezpublish/install/run.sh [hunk 4]
```diff
@@ -66,9 +66,14 @@ function patchSetupWizard
...
-        echo patching ....
-        patch -p0 < /setupwizard_ezstep_welcome.patch
-        cd - > /dev/null
+        if [ -d ezpublish_legacy ]; then
+            echo patching ....
+            patch -p0 < /setupwizard_ezstep_welcome.patch
+        else
+            echo "Warning : Skipping patching setup wizard. ezpublish_legacy/ do not seem to be present"
+        fi
+            cd - > /dev/null
```
Comment (andrerom) [inline dockerfiles/ezpublish/install/run.sh]: cs?

**Comment by andrerom** [comment]
+1

**Pull request merged**